### ISSUE TITLE
Header image has no alt tag

### ISSUE THREAD

**Issue by cmonagle** [opennorth/citizenbudgetapp.com#143]
Header image has no alt tag. should be populated with title.
Workaround (when hiding title)
```
/** Better for screen readers - header image has no alt tag**/
header h1 p {
  position: absolute;
  width: 1px;
  height: 1px;
  padding: 0;
  margin: -1px;
  overflow: hidden;
  clip: rect(0,0,0,0);
  border: 0;
}
```
